src/components/proposal/ProposalSummary.tsx

```diff
@@ -105,6 +105,9 @@ function ProposalReview({ confirm }: ProposalSummaryProps) {
           }
         }
       }
+      if (call?.submissionMessage != null) {
+        setSubmitButtonMessage(call.submissionMessage);
```

Comment:
Instead of overwriting the existing message (which dynamically says whether the proposal is editable after submission), I think we instead want to append a paragraph with this new text.

Reply:
Done